Gemfile.lock

```diff
@@ -1,7 +1,7 @@
 PATH
   remote: .
   specs:
-    devextreme-rails (22.1.3.0.6)
+    devextreme-rails (22.1.6.0.1)
```

Comment:
is the .6 tracking the underlying javascript version bump?

Reply:
Yip